ckan/saml2/who.ini

```diff
@@ -0,0 +1,38 @@
+[plugin:auth_tkt]
```

Comment:
You don't need to modify who.ini if you are going to use the saml2auth extension, since we don't use s2repoze in the new plugin. Also the s2repoze plugin of pysaml2 is not even maintained anymore, so it's not recommended.